se/se_epub_lint.py

```diff
@@ -2057,7 +2057,7 @@ def lint(self, skip_lint_ignore: bool, allowed_messages: List[str] = None) -> li
 					messages.append(LintMessage("s-098", "[xhtml]<header>[/] element with only one child.", se.MESSAGE_TYPE_WARNING, filename, [node.to_string() for node in nodes]))
 
 				# Check for a list of some common English loan words that used to be italicized as foreign, but are no longer
-				nodes = dom.xpath("/html/body//*[@data-css-font-style='italic' and re:test(., '^(fianc[eé]+|divorc[eé]+|menu|recherch[eé]|tour-de-force|outr[eé]|d[ée]but(ante)?|apropos|[eé]lite|prot[ée]g[ée]+|chef|salon|r[eé]gime|contretemps|[eé]clat|aides?|entr[ée]+)$')]")
+				nodes = dom.xpath("/html/body//*[@data-css-font-style='italic' and re:test(., '^(fianc[eé]+|divorc[eé]+|menu|recherch[eé]|tour-de-force|outr[eé]|d[ée]but(ante)?|apropos|[eé]lite|prot[ée]g[ée]+|chef|salon|r[eé]gime|contretemps|[eé]clat|aides?|entr[ée]+|s[eé]ances?|porti[eè]res?|dilettante|motifs?|t[eê]te-[aà]-t[eê]tes?|[eé]migr[eé]|ballerinas?|corv[eé]e|blas[eé]|fiasco(es|s)?|[eé]claircissements?|plaza|na[iï]fs?|poseuses?|bourgeoisies?|formula(e|s)?|causeuses?)$')]")
```

Comment:
Generally we only want to add loan words that are relatively common in English and are also commonly italicized in older books. Otherwise the regex could grow to enormous proportions but for very little practical utility. Of these I think we should only keep:

- blase
- tete-a-tete
- bourgeois (which also catches the plural)

The rest are rarely italicized so they are best kept as exceptions for the proofreader to find.

Reply:
IMO there's nothing strictly wrong with a large regex, though past a certain size you'd definitely want to split it into multiple lines.

I included "bourgeoisies" here since it's in MW, but I'll take it out for now.